content/documentation/pages/2-concepts/6-monitoring.md

```diff
@@ -12,18 +12,26 @@ The Micrometer instrumentation library powers the delivery of application metric
 
 The prebuilt applications are configured to support two of the most popular monitoring systems, Prometheus and InfluxDB. You can declaratively select which monitoring system to use.
 
-To help you get started monitoring Streams, Data Flow provides Grafana Dashboards you can install and customize for your needs.
+To help you get started monitoring Streams and Tasks, Data Flow provides Grafana Dashboards you can install and customize for your needs.
 
-The following image shows the general architecture of how applications are monitored:
+The following image shows the general architecture of how the Stream applications are monitored:
 
 ![Data Flow Stream Monitoring Architecture](images/micrometer-arch.png)
```

Comment:
Perhaps we can rename the file to be consistent with Task architecture's file name?